install.sh

```diff
@@ -10,6 +10,7 @@ PROJECT_NAME="arduino-cli"
 # BINDIR represents the local bin location, defaults to ./bin.
 EFFECTIVE_BINDIR=""
 DEFAULT_BINDIR="$PWD/bin"
+TEMPDIR="${TMPDIR:-${TEMP:-${TMP:-/tmp}}}"
```

Comment:
```suggestion
TMPDIR="${TMPDIR:-${TEMP:-${TMP:-/tmp}}}"
```
You're declaring the variable as `TEMPDIR` but using it as `TMPDIR`, without the letter `e`.

This breaks the script, I'll let you decide if you prefer `TEMPDIR` or `TMPDIR`. 
Using the correct variable name, your feature works as expected.

Looking forward to the quick patch. Thank you for your contribution! :nerd_face:

Reply:
Hello, thank you for noticing that! My system uses the TMPDIR variable, so it worked even with the wrong name, sorry. I've just amended the commit, so it should be fine now.